### PR TITLE
add toc and expose data sources pages

### PR DESCRIPTION
**Pull request by n-h-diaz**
will add redirects on website as follow up

### CODE REVIEW

File: datasets/biomedical.md [hunk 1]
```diff
@@ -5,8 +5,12 @@ nav_order: 3
 parent: Data Sources
 ---
 
+{:.no_toc}
 # Biomedical Data
 
+1. TOC
```
Comment (beets) [inline datasets/biomedical.md:11]: could we use disc bullets for these? so:
```suggestion
* TOC
```

**Review by beets — APPROVED** [review]
thanks - looks good! 

it could use a big of styling though.. how about:

```css
#markdown-toc {
    background: var(--light);
    padding: 1rem 2rem;
    border-radius: 5px;
}
```

File: datasets/biomedical.md (hunk 1 above)
Comment (n-h-diaz) [reply datasets/biomedical.md]: done

**Comment by n-h-diaz** [comment]
thanks for the review!

**Pull request merged**